float64[] attractor_position
float64 attractor_orientation
string pfield_id
---
bool success